classes/fsl-kernel-localversion.bbclass

```diff
@@ -40,4 +40,4 @@ do_preconfigure() {
 		printf "%s%s" +g $head > ${S}/.scmversion
 	fi
 }
-addtask preconfigure before do_configure after do_unpack do_patch
+addtask preconfigure before do_configure after do_kernel_metadata
```

Comment:
I believe you cannot remove the previous tasks as `kernel_metadata` is a `linux-yocto` specific task and we need to keep backward compatibility.

Reply:
@otavio thanks for the comment

Have modified that as you suggested in patch set V2.